### PR TITLE
Lazy load third party ai service custom ui

### PR DESCRIPTION
**Pull request by chetankothari**

### CODE REVIEW

File: src/components/ServiceDetails/AboutService/ServiceDemo/ThirdPartyAIService.js [hunk 1]
```diff
@@ -34,35 +29,24 @@ class ThirdPartyAIService extends Component {
       this.serviceRequestCompleteHandler
     );
     const { serviceSpecJSON, protoSpec } = await this.fetchServiceSpec(org_id, service_id);
-    this.fetchAIServiceComponent();
-    this.setState({ loading: false, serviceSpecJSON, protoSpec });
+    this.serviceSpecJSON = serviceSpecJSON;
+    this.protoSpec = protoSpec;
+    await this.fetchUserFeedback();
```
Comment (Vivek205) [inline src/components/ServiceDetails/AboutService/ServiceDemo/ThirdPartyAIService.js]: Need not to await this call.. Let it happen asynchronously

Comment (Vivek205) [reply src/components/ServiceDetails/AboutService/ServiceDemo/ThirdPartyAIService.js]: rest LGTM

File: src/SnetApp.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import SandboxApp  from "./sandbox/SandboxApp";
+import App from "./App";
+
+export default () => {
```
Comment (Vivek205) [inline src/SnetApp.js]: missing Lazy loading here

File: src/components/ServiceDetails/AboutService/ServiceDemo/ThirdPartyAIService.js (hunk 1 above)
Comment (chetankothari) [reply src/components/ServiceDetails/AboutService/ServiceDemo/ThirdPartyAIService.js]: Addressing this in the following PR which fixes the sandbox mode.

File: src/SnetApp.js (hunk 2 above)
Comment (chetankothari) [reply src/SnetApp.js]: Have lazy loading for service details. Don't think we need it here.

**Pull request merged**